common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -72,14 +77,32 @@ def assert_enrollment_status(
         response = self.client.post(url, json.dumps(data), content_type='application/json', **extra)
         self.assertEqual(response.status_code, expected_status)
 
-        if expected_status in [status.HTTP_200_OK, status.HTTP_200_OK]:
```

Comment:
I had to stare at it for a while to make sure I wasn't missing anything...¯\\(°_°)/¯